### PR TITLE
Added a bunch of sites.

### PR DESCRIPTION
**Pull request by sprinkledcheese**

### CODE REVIEW

**Comment by Gusted** [comment]
You visit a lot of sites...

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -468,6 +480,7 @@ viduro.xyz
 villains.fandom.com
 vimm.net
 vinesauce.com
+vipline.co.uk
```
Comment (Gusted) [inline src/config/dark-sites.config:483]: Vind the yellow pretty bright unsure about this one..

Comment (sprinkledcheese) [reply src/config/dark-sites.config:483]: Dark reader certainly makes a huge difference, will remove it.

**Comment by sprinkledcheese** [comment]
> You visit a lot of sites...

Yes and no. Many are recommended by friends and acquaintances.

**Comment by Gusted** [comment]
Ah that make more sense 😂

**Pull request merged**